docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -1,120 +1,96 @@
+```
 
-| Name | Use for... | User Access Requirements | Cost |
-| ---- | ----------- | ----------- | ----- |
-| [bcgov](#bcgov) | Repositories should be hosted in this open repo unless you have strong rationale from your MISO otherwise. | GitHub ID | Free |
-| [bcgov-c](#bcgov-c) | Code you would like to temporarily host in a closed repo, before moving into the bcgov org. | GitHub ID | Covered by OCIO for up to 12 months |
+Both organizations require users to enable [Single Sign-on](## Single sign-on).
 
 ### `bcgov`
 
-The `bcgov` organization contains public repositories that hold open-source code or public documents developed by B.C. government teams. This organization is **public**.
+The `bcgov` organization contains public repositories that hold open-source code or public documents developed by B.C. government teams. This organization is **public**.  To align with our Digital Principles, repositories should be hosted in this open repo, unless you have strong rationale from your Ministry Security Information Officer. 
 
-* Product teams working in the B.C. Gov Private Cloud PaaS OpenShift platform should use this organization unless they are given permission to work in [`bcgov-c`](#bcgov-c).
-* Any member of the `bcgov` organization can create repositories.
-* Existing members of the `bcgov` organization can invite new users to join the organization using the [Just Ask! tool](https://just-ask.developer.gov.bc.ca/). (***Users requesting membership to `bcgov` for themselves will be reviewed and may be declined***) 
+* Product teams working in the B.C. Government Private Cloud PaaS OpenShift platform should use this organization, unless they are given permission to work in [`bcgov-c`](#bcgov-c)
+* Any member of the `bcgov` organization can create repositories
 
 ### `bcgov-c`
 
-The `bcgov-c` organization stores temporary (up to 12 months), private repositories with closed-source code and private documents. Closed-source projects must be moved to the `bcgov` organization at the end of the 12 months. This repository is **private**.
-
-* Use this repository if you need a temporary location for code while you collect approvals to make the code public. You must commit to making the code public in the future in order to use this repository.
-* Only the Developer Experience team can create repositories in this organization. You can request a repository by [submitting a request](https://github.com/BCDevOps/devops-requests/issues/new?assignees=MonicaG%2C+oomIRL&labels=github-repo%2C+pending&projects=&template=github_repo_request.md&title=).
+The `bcgov-c` organization stores permanent, private repositories with confidential source code and documents. This repository is **private**.
 
-Your product team can only have a **permanent**, private repository in `bcgov-c` if it is a GitOps repository with ArgoCD manifests. 
+* Use this repository if you need a permanent location for code, or if you're collecting approvals to make the code public.
```

Comment:
collecting approvals could maybe be "working towards"? there's more work involved usually than just approvals unfortunately